data/helpers.d/string

```diff
@@ -9,3 +9,18 @@ ynh_string_random() {
       | tr -c -d 'A-Za-z0-9' \
       | sed -n 's/\(.\{'"${1:-24}"'\}\).*/\1/p'
 }
+
+# Substitute/replace a string by another in a file
+#
+# usage: ynh_replace_string string_to_find replace_string target_file
+# | arg: string_to_find - String to be searched and replaced in the file
+# | arg: replace_string - String that will replace matches
+# | arg: target_file - File in which the string will be replaced.
+ynh_replace_string () {
+	delimit=@
+	match_char=${1//${delimit}/"\\${delimit}"}	# Escape the delimiter if it's in the string.
```

Comment:
to be fully consistent with the function renaming (with which I agree), you could rename these two variables as well.